python/lsst/meas/algorithms/convertRefcatManager.py

```diff
@@ -454,3 +454,35 @@ def gaiaFluxToFlux(flux, zeroPoint):
         result['phot_rp_mean_fluxErr'] = result['phot_rp_mean_flux'] / input['phot_rp_mean_flux_over_error']
 
         return result
+
+class ConvertGaiaXpManager(ConvertRefcatManager):
+    """Special-case convert manager to deal with refcats that have no MagErrors
+    """
+    def _getFluxes(self, inputData):
+        """Compute the flux fields that will go into the output catalog.
```

Comment:
I would leave the whole docstring for this method empty, so that it uses the parent class docstring when building the docs, since the inputs/outputs are the same.